way-cooler/README.md

```diff
@@ -0,0 +1,132 @@
+# Way Cooler
+
+[![Gitter](https://badges.gitter.im/Immington-Industries/way-cooler.svg)](https://gitter.im/Immington-Industries/way-cooler?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![Crates.io](https://img.shields.io/crates/v/way-cooler.svg)](https://crates.io/crate/way-cooler)
+[![Build Status](https://travis-ci.org/Immington-Industries/way-cooler.svg?branch=master)](https://travis-ci.org/Immington-Industries/way-cooler)
+[![Coverage Status](https://coveralls.io/repos/github/Immington-Industries/way-cooler/badge.svg)](https://coveralls.io/github/Immington-Industries/way-cooler)
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/Immington-Industries/way-cooler/)
+
+Way Cooler is a customizable tiling window manager written in [Rust][] for [Wayland][wayland].
+
+
+# Development
+
+Way Cooler is currently in alpha. The core features have been added and it is in a usable state, but more work is needed to
+make it user friendly. Here's an example where we run Way Cooler within i3. Everything within the wlc-x11 window is Way Cooler:
+
+
+[![way-cooler demonstration](http://i.imgur.com/UiJbpiv.png)](https://www.youtube.com/watch?v=I2FO5dnOBb0)
+
+## Motivation
+
+We wanted to get experience with Rust and we found current X11 window managers to not have all the features we wanted.
+
+Currently there are very few fully-featured tiling window managers in the Wayland ecosystem, as most of the effort has been porting Gnome and KDE over. Although Wayland is still in early-stage development
+and is not backwards compatible with existing X11 tools, we wanted to put our stake in and provide for current tiling window manager users in the future.
+
+We take a lot of inspiration from current window managers (namely [i3][] and [awesome][]) but our goal is to exist as a unique alternative.
+
+
+## Current Features
+- i3-style tiling
+  * Horizontal/vertical layouts
+  * Nest containers with different layouts
+- Client application support via an IPC
+  * See an example application [here](https://github.com/Immington-Industries/Way-Cooler-Example-Clients). It displays the tree in a somewhat organized format, and is actually really helpful for both debugging the tree and understanding how subcontainers work.
+  * Enables dynamic configuration at runtime, without having to reload a configuration file
+  * Allows extensions of the window manager to exist as separate programs talking over the IPC
+- A Lua environment designed to make extending Way Cooler simple and easy
+  * Lua is the configuration format, allowing the user to enhance their window manager in any way they want.
+  * Utilities library included to aid communicating with Way Cooler
+- X programs supported through XWayland
+
+## Planned Features
+
+- i3 tabbed/stacked tiling
+- Floating windows
```

Comment:
You have the honor of removing this from the "planned features" list. I dunno what improvements you wanted to make to floating windows still but we definitely have them.